refinery/tool_manager/migrations/0017_tooldefinition_extra_directories.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tool_manager', '0016_auto_20170424_1753'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='tooldefinition',
+            name='extra_directories',
+            field=models.CharField(max_length=500, blank=True),
+            preserve_default=True,
+        ),
+    ]
```

Comment:
@mccalluc I think pluralization is okay for this implementation, but I've been thinking about this precisely for the items we currently have to `ast.literal_eval()`. 

We can't natively store JSON db-side until we get to a later Django version, but I think it would be adequate to have one `TextField` that I store a Json string in which can deserialize and interact with later.